base_edi/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "name": "Base EDI",
     "summary": """Base module to aggregate EDI features.""",
     "version": "12.0.1.0.1",
-    "development_status": "Alpha",
+    "development_status": "Beta",
```

Comment:
what motivates this change?

Reply:
The **edi_oca** module has a development_status of "beta" and depends on a module with a development_status of "alpha" and got a warning in test. And in base_edi v14 there is a commit to move it (https://github.com/OCA/edi/commit/e5db4e03896e9db968e137b9772d0f4db957981c)

Reply:
It's so little that I'll fix it quickly. Or should I cherry-pick it?

Reply:
keep it as is, but make it explicit in PR description why you had to do this